src/GEXF/Gexf.jl

```diff
@@ -50,6 +51,7 @@ function savegexf(io::IO, g::LightGraphs.AbstractGraph, gname::String)
     return 1
 end
 
-savegraph(io::IO, g::AbstractGraph, gname::String, ::GEXFFormat) = savegexf(io, g, gname)
```

Comment:
I think we still need this since without it we won't be able to save non-Metagraphs.